integration/common-provision.sh

```diff
@@ -24,13 +24,11 @@ go get github.com/tools/godep
 # Build p2.
 cd $GOPATH/src/github.com/square/p2
 godep go install ./...
+cp $GOPATH/bin/p2-exec /usr/local/bin
 
 # Install ruby + rake
 sudo yum install -y ruby rubygem-rake
 
-# Install cgexec
-sudo yum install -y libcgroup libcgroup-tools
```

Comment:
it was only needed as a dependency of cgexec (libcgroup-tools) and in fact probably we never had to explicitly put it there at all. (I don't know how yum works so idk)